CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.66.3 10-28-2024
+* Update Fallback Fonts from sans-serif to serif in Rollingstone 2022
+
```

Comment:
This should go under the unplublished changes heading.

Reply:
@rsalvucci I didn't see this but ended up fixing it in the release branch